packages/clarity-lang/clarity-lang.0.1.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Clarity smart contract parser and AST"
+description:
+  "Clarity.ml parses Clarity smart contracts into an abstract syntax tree."
+maintainer: ["Arto Bendiken <arto@bendiken.net>"]
+authors: ["Arto Bendiken <arto@bendiken.net>"]
+license: "Unlicense"
+homepage: "https://github.com/weavery/clarity.ml"
+bug-reports: "https://github.com/weavery/clarity.ml/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "alcotest" {with-test}
+  "menhir" {>= "20200624"}
+  "num" {>= "1.3"}
```

Comment:
```suggestion
  "num" {>= "1.3"}
  "cppo" {build}
```
The cppo package is used for building

Reply:
Can you please upstream this change?

Reply:
@mseri Thanks, indeed needs to be added!

Reply:
https://github.com/weavery/clarity.ml/commit/ab2c04273fdbce856da7fb55161598e92de5e43d